### PR TITLE
chore: add cpu benchmark, languaje for diferent networks and basic na…

### LINKED ISSUE #26: CPU Benchmarking
There is a collection of [smart contracts and utilities for EOSIO](https://github.com/AlohaEOS/eos-mechanics) that can be used 

## CPU Benchmark
This CPU speed metric uses the CPU time consumed by calculating the Mersenne prime numbers as a benchmark. Prime number calculation is an industry standard for measuring CPU performance and uses code operations that are common in software development.

## On Chain Monitoring

The metrics are generated by smart contracts located under the eosmechanics account. They are executed during the schedule of each block producer and records the execution times using the cpu_usage_us field included in all network transactions. The data will be publicly available for analysis and we encourage you to do so to help identify problems and improve the performance of block producers.

Examples of tools that use this data:

[EOS Block Producer Benchmark](https://www.alohaeos.com/tools/benchmarks)s
[Block Producer Performance
](https://labs.eostitan.com/#/block-producer-performance)

### LINKED ISSUE #128: Set language file for different networks
Different networks refer to the same thing differently

for example "Block Producers" are referred to as "Validators"

This can be done by setting network name in env variable.

### LINKED ISSUE #130: remove nodemon

### PR DESCRIPTION
**Pull request by adriexnet**
### GH Issue
- Resolve #26
- Resolve #128 
- Resolve #130

### What does this PR do?
- Implement CPU benchmark
- Allow specific languaje file per network
- Remove nodemon

### Steps to test
1. set a valid value for `REACT_APP_USELANGUAGE_SUFIX`
3. yarn start
4. naviagte to http://localhost:3000/dashboard/network
5. change the language

#### CheckList
- [x] Follow proper Markdown format
- [x] The content is adequate
- [x] The content is available in both english and spanish
- [x] I Ran a spell check

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
Awesome!

File: webapp/src/i18n.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import i18n from 'i18next'
 import LanguageDetector from 'i18next-browser-languagedetector'
 import { initReactI18next } from 'react-i18next'
 import resources from './language'
-
+console.log(resources)
```
Comment (tetogomez) [inline webapp/src/i18n.js:5]: this console is necessary?

**Review by tetogomez — APPROVED** [review]
LGTM

File: webapp/src/language/en.json [hunk 2]
```diff
@@ -101,7 +102,9 @@
     "pingFromCostaRica": "Ping from Costa Rica",
     "serverVersion": "Server version",
     "secondsAgo": "s ago",
-    "lastTimeChecked": "Last Time Checked"
+    "lastTimeChecked": "Last Time Checked",
+    "healthStatus": "Health Status",
+    "cpuUsage": "CPU Usage"
```
Comment (xavier506) [inline webapp/src/language/en.json:107]: CPU Benchmark

File: webapp/src/language/en.json [hunk 3]
```diff
@@ -110,5 +113,12 @@
     "apiNotResponding": "API Not Responding",
     "bpJsonFound": "BP JSON found",
     "noBpJsonFound": "No BP JSON found"
+  },
+  "dashboardNetwork": {
+    "htmlTitle": "Network Info - Dashboard",
+    "title": "Network Info",
+    "chainLimits": "Chain Limits",
+    "cpuLimitPerBlock": "CPU LIMIT PER BLOCK",
+    "netLimitPerBlock": "NET LIMIT PER BLOCK"
```
Comment (xavier506) [inline webapp/src/language/en.json:122]: Chain ID

File: webapp/src/language/es.lacchain.json [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    "nodeDistribution": "Distribución de nodos",
+    "rewardDistribution": "Distribución de recompensas",
+    "nodeInfo": "Actualizar información del nodo",
+    "about": "Acerca de",
+    "ricardianContract": "Contrato ricardiano",
+    "help": "Ayuda"
+  },
+  "dashboardHome": {
+    "currentProducer": "Productor actual",
+    "headBlock": "Bloque de la cabeza",
```
Comment (xavier506) [inline webapp/src/language/es.lacchain.json:23]: Ultimo Bloque

**Pull request merged**